spock-core/core.gradle

```diff
@@ -21,6 +21,7 @@ configurations {
 }
 
 dependencies {
+  api libs.geantyref
```

Comment:
I think this should be an `implementation` dependency and moved after the `api` elements